### PR TITLE
More informative docstrings in the random module

### PR DESCRIPTION
**Pull request by rhettinger**
Add the expected mean and variance to the docstrings.  It helps with understanding the parameters which is important because conventions vary.  Also, it would save someone like me from having to look these up periodically.

### CODE REVIEW

File: Lib/random.py [hunk 1]
```diff
@@ -492,7 +492,11 @@ def choices(self, population, weights=None, *, cum_weights=None, k=1):
     ## -------------------- real-valued distributions  -------------------
 
     def uniform(self, a, b):
-        "Get a random number in the range [a, b) or [a, b] depending on rounding."
+        """Get a random number in the range [a, b) or [a, b] depending on rounding.
+
+        E[X] = (a + b) / 2
```
Comment (AA-Turner) [inline Lib/random.py]: Should we note that this is the mean, for users that might be unfamilliar with the expectation operator?

```suggestion
        (mean): E[X] = (a + b) / 2
```

**Comment by AA-Turner** [comment]
I think it may also be worth updating the documentation to include this information, to quickly refer to it without opening an interpreter.

A

**Comment by rhettinger** [comment]
FWIW here's session showing expected versus actual:

```
>>> from random import uniform, triangular, expovariate, gammavariate, betavariate, binomialvariate
>>> from statistics import mean, variance
>>> n = 10 ** 6

>>> a, b = 75, 20
>>> mean(uniform(a, b) for i in range(n)), (a + b) / 2
(47.514550556196895, 47.5)
>>> variance(uniform(a, b) for i in range(n)), (b - a) ** 2 / 12
(252.13423065351284, 252.08333333333334)

>>> low, high, mode = 100, 150, 110
>>> mean(triangular(low, high, mode) for i in range(n)), (low + high + mode) / 3
(120.0327029921039, 120.0)
>>> variance(triangular(low, high, mode) for i in range(n)), (low**2 + high**2 + mode**2 - low*high - low*mode - high*mode) / 18
(116.58565949632437, 116.66666666666667)

>>> lambd = 0.625
>>> mean(expovariate(lambd) for i in range(n)), 1 / lambd
(1.6019558105152012, 1.6)
>>> variance(expovariate(lambd) for i in range(n)), 1 / lambd ** 2
(2.5613327986115277, 2.56)

>>> alpha = 1.5; beta = 0.625
>>> mean(gammavariate(alpha, beta) for i in range(n)), alpha * beta
(0.9361273191982766, 0.9375)
>>> variance(gammavariate(alpha, beta) for i in range(n)), alpha * beta ** 2
(0.5886486276596401, 0.5859375)

>>> alpha = 5; beta = 8
>>> mean(betavariate(alpha, beta) for i in range(n)), alpha / (alpha + beta)
(0.38450002539757394, 0.38461538461538464)
>>> variance(betavariate(alpha, beta) for i in range(n)), alpha * beta / ((alpha + beta)**2 * (alpha + beta + 1))
(0.016937933060186688, 0.0169061707523246)

>>> n = 25; p = 0.375
>>> mean(binomialvariate(n, p) for i in range(n)), n * p
(9.28, 9.375)
>>> variance(binomialvariate(n, p) for i in range(n)), n * p * (1 - p)
(5.333333333333333, 5.859375)
```

**Review by AA-Turner — APPROVED** [review]

**Comment by AA-Turner** [comment]
@rhettinger -- what are your thoughts on including these data in the documentation for ``random``? I'm happy to prepare a PR if you don't object.

A

**Comment by rhettinger** [comment]
@AA-Turner I don't think this needs to be in the main docs. It is more of an on-line reference.

**Pull request merged**